aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Docker images for ASP.NET Core
+author: tdykstra
+description: Learn how to use the published .NET Core Docker images from the Docker Registry. Pull images and build your own images.
+ms.author: tdykstra
+ms.custom: mvc
+ms.date: 04/09/2019
+---
+
+# Docker images for ASP.NET Core
+
+This tutorial shows how to run an ASP.NET Core app in Docker containers.
+
+In this tutorial, you:
+> [!div class="checklist"]
+> * Learn about Microsoft .NET Core Docker images 
+> * Download an ASP.NET Core sample app
+> * Run the sample app locally
+> * Run the sample app in Linux containers
+> * Run the sample app in Windows containers
+> * Build and deploy manually
+
+## ASP.NET Core Docker images
+
+For this tutorial, you download an ASP.NET Core sample application and run it in Docker containers. The sample works with both Linux and Windows containers.
```

Comment:
For this tutorial, you download an ASP.NET Core sample app and run it in Docker containers. The sample works with both Linux and Windows containers.